torch/distributed/elastic/utils/distributed.py

```diff
@@ -16,7 +16,6 @@
 log = get_logger()
 
 _ADDRESS_IN_USE = "Address already in use"
-_CONNECT_TIMEOUT = "connect() timed out."
```

Comment:
Looks like `test_tcp_store_timeout_set` reads the `connect() timed out` message to determine whether or not to retry on failure. May also need to update this test to prevent it from being flaky.